exploration/expression-attributes.md

```diff
@@ -176,20 +191,16 @@ including expressions without an annotation.
 
 Attributes are distinct from function options.
 
-Common attributes are defined by the MF2 specification
-and must be supported by all implementations.
+Common options or attributes should work the same way in different functions.
+
+Special options or attributes should not conflict with other option names.
```

Comment:
Special, because that can account for implementations-specific things that may be "common" within their scope, but not universally. So e.g. `@amzn:marketplace=US` or `amzn:marketplace=US`.